### PR TITLE
Moreover unit test

### PR DESCRIPTION
**Pull request by rbnks**
created unit test and updated dependency version.

### CODE REVIEW

File: streams-contrib/streams-provider-moreover/src/test/java/org/apache/streams/data/moreover/MoreoverResultTest.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+/**
+ *
+ */
+public class MoreoverResultTest {
+
+
+    @Test
+    public void testMoreoverResultProcess() {
+        MoreoverResult result = new MoreoverResult("", getResultString("/MoreoverXMLResponse.xml"), 0, 0);
```
Comment (sieverssj) [inline streams-contrib/streams-provider-moreover/src/test/java/org/apache/streams/data/moreover/MoreoverResultTest.java]: I'm not seeing this file in the PR.  Does it already exist?

File: pom.xml [hunk 2]
```diff
@@ -183,7 +183,7 @@
         <!-- Library Dependency Versions -->
         <org.osgi.service.http.port>8080</org.osgi.service.http.port>
         <org.osgi.service.http.port.secure>8443</org.osgi.service.http.port.secure>
-        <jackson.version>2.3.2</jackson.version>
+        <jackson.version>2.5.0</jackson.version>
```
Comment (robdouglas) [inline pom.xml]: Why did we need to bump the Jackson version?

File: streams-contrib/streams-provider-moreover/src/test/java/org/apache/streams/data/moreover/MoreoverResultTest.java (hunk 1 above)
Comment (rbnks) [inline streams-contrib/streams-provider-moreover/src/test/java/org/apache/streams/data/moreover/MoreoverResultTest.java]: This was a new test created.

Comment (sieverssj) [inline streams-contrib/streams-provider-moreover/src/test/java/org/apache/streams/data/moreover/MoreoverResultTest.java]: MoreoverXMLResponse.xml, I mean.

File: pom.xml (hunk 2 above)
Comment (rbnks) [inline pom.xml]: iron.io has version 2.5.0 in its pom.  and it causes a version conflict when parsing xml

File: streams-contrib/streams-provider-moreover/src/test/java/org/apache/streams/data/moreover/MoreoverResultTest.java (hunk 1 above)
Comment (rbnks) [inline streams-contrib/streams-provider-moreover/src/test/java/org/apache/streams/data/moreover/MoreoverResultTest.java]: thats not new.  only MoreoverXMLResponse2.xml is new.

**Comment by sieverssj** [comment]
:+1:

**Pull request merged**